components/threatconnect.js

```diff
@@ -253,11 +254,52 @@ polarity.export = PolarityComponent.extend({
 
       Ember.set(caseToUpdate, '__newAttributes', updatedAttributes);
     },
+    cancelRemoveDescription(indicatorId) {
+      const path = `indicators.${indicatorId}.indicator.__newCase`;
+      this.set(`${path}.__showDescriptionModal`, null);
+    },
+
+    confirmRemoveDescription(indicatorId) {
+      const path = `indicators.${indicatorId}.indicator.__newCase`;
+      const modalData = this.get(`${path}.__showDescriptionModal`);
```

Comment:
@ahaias-tc  I would rename `showDescriptionModal` to more accurately describe what it is.  The current naming suggests this value is a boolean but it looks like it's actually an object based on usage here.  It also doesn't seem to have anything to do with a "modal".

I think you want to make it clear this is related to workflow implementation maybe something like `selectedWorkflowData`.